packages/core/admin/admin/src/components/Dashboard/EEStatistics.tsx

```diff
@@ -0,0 +1,63 @@
+import { Grid, Typography } from '@strapi/design-system';
+import { formatDistanceToNow } from 'date-fns';
+import { useIntl } from 'react-intl';
+
+import { BigNumber as BigNumberChart } from './Charts/BigNumber';
+import { Audit as DashboardAudit } from './Charts/Lists/Audit';
+
+import type {
+  EEStatistics as EEStatisticsType,
+  Release,
+  Activity,
+} from '../../../../server/src/services/statistics';
+
+interface EEStatisticsProps {
+  statistics: EEStatisticsType;
+}
+
+export const EEStatistics: React.FC<EEStatisticsProps> = ({ statistics }) => {
+  const { formatMessage } = useIntl();
+
+  const { upcomingReleases, upcomingReleasesCount, latestActivities, workflowsCount } = statistics;
+
+  if (
+    !upcomingReleases.length &&
+    !latestActivities.length &&
+    !workflowsCount &&
+    !upcomingReleasesCount
+  )
+    return;
+  return (
+    <>
+      <Typography color="neutral0" fontSize={3} fontWeight="bold">
+        {formatMessage({
+          id: 'app.components.HomePage.dashboard.content-type.overview.labelfd',
+          defaultMessage: 'Advanced features',
+        })}
+      </Typography>
+      <Grid.Root gap={6} marginTop={4} marginBottom={8}>
+        {upcomingReleasesCount && (
```

Comment:
Don't use a number as a conditional or you will just get a 0 and react will render a zero.

![image](https://github.com/user-attachments/assets/ba5ec538-15e0-4284-8191-977274c395dd)